packages/teams-js/test/private/privateAPIs.spec.ts

```diff
@@ -206,7 +206,7 @@ describe('AppSDK-privateAPIs', () => {
     await initPromise;
 
     // Now the getContext call should have been dequeued
-    expect(utils.messages.length).toBe(2);
+    expect(utils.messages.length).toBe(3);
```

Comment:
>3 [](http://example.com/codeflow?start=39&length=1)

Are all these number incrementing changes required because now a new message is being sent in this flow (to check the origin with the host)?

If so, this model in the unit tests doesn't exactly seem sustainable (e.g., these unit tests are assuming that they know how many messages get sent out before them).

I'm not saying you have to fix this now with this change, but it would be good to file a bug to ensure we update these to make fewer assumptions about how the infrastructure is working.